phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/library/LibraryController.java

```diff
@@ -95,6 +104,34 @@ public LibraryController(ViewLibrary viewLibrary, ConsoleController consoleContr
         });
     }
 
+    /**
+     * creates a shortcut for a given executable
+     * @param executable executable file (WineShortcut will search for this file in the given prefix)
+     */
+    private void createShortcut(File executable) {
+        // get container
+        // TODO: smarter way using container manager
+        final String executablePath = executable.getAbsolutePath();
+        final String pathInContainers = executablePath.replace(containersPath, "");
+        final String[] split = pathInContainers.split("/");
+        final String engineContainer = split[0];
+        final String engine = (Character.toUpperCase(engineContainer.charAt(0)) + engineContainer.substring(1))
+                .replace("prefix", "");
+        final String container = split[1];
+
+        final InteractiveScriptSession interactiveScriptSession = scriptInterpreter.createInteractiveSession();
+
+        interactiveScriptSession.eval("include([\"Engines\", \"" + engine + "\", \"Shortcuts\", \"" + engine + "\"]);",
+                ignored -> interactiveScriptSession.eval("new " + engine + "Shortcut()", output -> {
+                    final ScriptObjectMirror shortcutObject = (ScriptObjectMirror) output;
+                    shortcutObject.callMember("name", executable.getName());
+                    shortcutObject.callMember("search", executable.getName());
+                    shortcutObject.callMember("prefix", container);
+                    shortcutObject.callMember("create");
+                }, e -> Platform.runLater(() -> new ErrorMessage(tr("Error while creating shortcut"), e))),
```

Comment:
Can't we move this (`e -> Platform.runLater(() -> new ErrorMessage(tr("Error while creating shortcut"), e))`) to another method called `defaultErrorHandler` or something?